packages/react-router-dom/modules/Router.js

```diff
@@ -1 +1 @@
-export { Router as default } from 'react-router'
+export { default } from 'react-router/Router'
```

Comment:
(Just picking one of these at random, but this applies to all the re-exports).

Unfortunately, in the packaged form, the ES module versions of these files actually live in `/es/`. So to get tree shaking benefits beyond these exports, you would actually have to link to `react-router/es/Router`. But that would break non-ES-aware bundlers. 

I'm not sure if there's a good option here. If you import directly from `react-router/es/Router`, that is the best way to get tree-shaking benefits. Perhaps we should remove the re-exports entirely?